### PR TITLE
Task #1579 :  Added Dune Dependancy Copy Button

### LINKED ISSUE #1579: Add a line for the dune-project file to the package overview page
# Task

On the package overview pages https://ocaml.org/p/irmin/latest (template at `package_overview.eml`), there is a button to copy the `opam install` command for the package. It would be nice to also have a button to copy the line you need to add to the `dune-project` file to add the dependency to your Dune project.

Screenshot 2023-10-05 at 12-03-34 irmin 3 8 0 (latest) · OCaml Package

Add another copyable code snippet below the existing one that contains the the line that needs to be placed in the `dune-project` file, could separate this with another heading "Dune Dependency".

Could look like this:

Screenshot from 2023-10-05 12-14-21

### PR DESCRIPTION
**Pull request by henilGondalia**
Resolves #1579.

I have addressed the issue #1579. Here's a summary of the changes:

ocaml 2023-10-09 214303

Please review the changes and let me know if any further adjustments are needed. Thank you!

### CODE REVIEW

File: src/ocamlorg_frontend/pages/package_overview.eml [hunk 1]
```diff
@@ -49,13 +49,20 @@ in
     <%s package.synopsis %>
   </div>
 
-  <h2 class="inline-flex items-center text-lg font-medium text-default">
+  <h2 class="inline-flex items-center text-lg font-medium text-default ">
```
Comment (SaySayo) [inline src/ocamlorg_frontend/pages/package_overview.eml]: There's an extra whitespace here. It is not needed.

**Comment by SaySayo** [comment]
Thank you for your contribution. The changes looks good to me!

**Review by SaySayo — APPROVED** [review]

File: src/ocamlorg_frontend/pages/package_overview.eml [hunk 2]
```diff
@@ -49,13 +49,20 @@ in
...
   </h2>
 
   <%s! Copy_to_clipboard_snippet.render ~id:"install-package" ~class_:"whitespace-nowrap" ("opam install " ^ package.name ^ "." ^ specific_version) %>
 
+  <h2 class="inline-flex items-center text-lg font-medium text-default mt-8">
+      <%s! Icons.command_line "mr-4 h-6 w-6 text-primary-600" %>
+      Dune Dependency
+  </h2>
+
+  <%s! Copy_to_clipboard_snippet.render ~id:"dune-dependency-package" ~class_:"whitespace-nowrap" (" (" ^ package.name ^ " (= " ^ specific_version ^ ") )" ) %>
```
Comment (sabine) [inline src/ocamlorg_frontend/pages/package_overview.eml]:
```suggestion
  <%s! Copy_to_clipboard_snippet.render ~id:"dune-dependency-package" ~class_:"whitespace-nowrap" (" (" ^ package.name ^ " (= " ^ specific_version ^ "))" ) %>
```

would also omit this whitespace

Comment (henilGondalia) [reply src/ocamlorg_frontend/pages/package_overview.eml]: Sure I will omit both white space

**Review by SaySayo — APPROVED** [review]

**Comment by sabine** [comment]
Thanks @henilGondalia! :sparkles:

**Comment by henilGondalia** [comment]
> Thanks @henilGondalia! ✨

My pleasure 😊

**Pull request merged**